csrc/optimization/alias_analysis.cpp

```diff
@@ -392,7 +392,8 @@ void AliasAnalysisResult::finalize(Fusion* fusion) {
 
 Layout AliasAnalysisResult::preferredLayout(const Val* v) const {
   const TensorView* tv = dynamic_cast<const TensorView*>(v);
-  NVF_CHECK(
+  NVF_ERROR(v != nullptr); // Just to make clang-tidy happy.
```

Comment:
nitpick: remove this and later in the error message we can do `v != nullptr ? "nullptr" : v->toSTring()`.
Does that keep clang happy as well?

Reply:
Done.